docs/products/opensearch/howto/cpu_alert.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
This file should be in the top-level `code` folder, and then included from there. If you need an example, all the other code includes should follow that pattern

Reply:
fixed it!